test/e2e/states.go

```diff
@@ -2,16 +2,36 @@ package e2e
 
 import (
 	"fmt"
+	"strings"
+	"text/template"
+
+	. "github.com/onsi/gomega"
 
 	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
 )
 
-func ethernetNicUp(nicName string) nmstatev1alpha1.State {
-	return nmstatev1alpha1.NewState(fmt.Sprintf(`interfaces:
-  - name: %s
+func ethernetNicsState(states map[string]string) nmstatev1alpha1.State {
```

Comment:
Could you please just create multiple policies, one per each NIC? So we decrease the number of helper function, plus reading the template is hard.

Reply:
Well we have around 24 test now we do 24 policy apply to do that if we do per nic that would be 72 so we open the door to something going bad network wise and is less atomic.